cms/search/tests/test_api_views.py

```diff
@@ -15,7 +15,7 @@
 from cms.themes.tests.factories import ThemePageFactory
 from cms.topics.tests.factories import TopicPageFactory
 
-RESOURCE_ENDPOINT = "/v1/resources/"
+RESOURCE_ENDPOINT = "/v1/resources"
```

Comment:
I assumed here this endpoint should also not have a trailing slash, but can be added to exceptions.

Reply:
@MebinAbraham